mocks/modules/task/cherami/Client.go

```diff
@@ -0,0 +1,378 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
can't find a way to generate them with a custom package name yet. might send a later PR if I find a way.